engine/interpreter/src/main/scala/pl/touk/nussknacker/engine/splittedgraph/part.scala

```diff
@@ -24,6 +24,12 @@ object part {
     override type T = CustomNode
   }
 
+  case class EndingCustomNodePart(node: EndingNode[CustomNode]) extends SubsequentPart {
```

Comment:
Why not to use just `CustomNodePart` with `nextParts = empty` and `ends = empty`?

Reply:
I reused `CustomNodePart` at split stage, but  I'll leave `EndingCustomNodePart` at compiled graph structure till we make a try with flink runtime.

Reply:
After all I also reused CustomNodePart in compile graph.